src/perf/tracepoint/format.cpp

```diff
@@ -98,6 +98,29 @@ void EventFormat::parse_format_line(const std::string& line)
     }
 }
 
+std::vector<std::string> EventFormat::get_tracepoint_event_names() try
```

Comment:
You're missing a pair of curly braces.

Reply:
Today I learned that this is actually a valid syntax construct: a [function-try-block](http://en.cppreference.com/w/cpp/language/function-try-block). Will change nonetheless.